samples/nrf9160/slm_shell/README.rst

```diff
@@ -40,6 +40,9 @@ The following table shows how to connect PCA10056 UART_1 to nRF9160 UART_2 for c
    * - GPIO GND
      - GPIO GND
 
+.. note::
+   SW9 on nRF9160 DK (VDD IO) should be set to 3V to work with the nRF52 series DK.
```

Comment:
```suggestion
   The GPIO output level on the nRF9160 side must be 3 V to work with the nRF52 series DK.
   You can set the VDD voltage with the **VDD IO** switch (**SW9**).
```
Taking the chance to align with the formatting of the note in the [LTE modem](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/serial_lte_modem/doc/slm_description.html#connecting-with-an-external-mcu) description.